### ISSUE TITLE
Schaduling doesn't work properly

### ISSUE THREAD

**Issue by ahangarha** [NicolasConstant/sengi#216]
I have made a schedule post for 11:53 AM. But it was not published at all. It is still in the scheduled statuses.

![image](https://user-images.githubusercontent.com/11241315/74632906-177cb780-517e-11ea-8e70-76a1b5fb30b4.png)

I guess there is an issue with timezone. Is it so? I wait for it to be published and see if there is any specific time difference or not.

**Comment by NicolasConstant** [comment]
I just tested it (I'm on the Canadian east-coast) and the scheduled status was published at the right time BUT the scheduled panel didn't refresh properly, is it the same behavior on your side?

**Comment by stom79** [comment]
I noticed that scheduling from server side doesn't work properly for all instances. They still appear like scheduled but were never sent. 
Tested on fediplan or Fedilab scheduling (server side).
I tested on an instance where it fails each time and it was not related to the timezone.

**Comment by ahangarha** [comment]
I have no issue with scheduling on Fedilab.

**Comment by stom79** [comment]
Did you pick up server side too on Fedilab?

**Comment by ahangarha** [comment]
I think I do it from device. If Sengi does scheduling from server side, I will try it on Fedilab to see the result

**Comment by NicolasConstant** [comment]
Yes, Sengi is using the server side's scheduling, I wasn't aware that some instance might be problematic... :/

**Comment by ahangarha** [comment]
Then perhaps we need to show the option if scheduling is allowed by the instance.

**Comment by stom79** [comment]
Unfortunately, there is no way to know it. I think that's a server side configuration.
I didn't investigate much more (ask the admins or check the doc about what is needed to make it work).
